.gitignore

```diff
@@ -6,6 +6,7 @@
 /lib/elixir/src/*_parser.erl
 /lib/elixir/src/elixir.app.src
 /lib/elixir/test/ebin
+/lib/elixir/_build
```

Comment:
To test, you can run "make test_elixir" from the root.



*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*

On Sat, Sep 5, 2015 at 12:36 PM, Andrew Forward <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/elixir-lang/elixir/pull/3673#discussion_r38809631>:
>
> > @@ -6,6 +6,7 @@
> >  /lib/elixir/src/*_parser.erl
> >  /lib/elixir/src/elixir.app.src
> >  /lib/elixir/test/ebin
> > +/lib/elixir/_build
>
> Ahh, ok. I think tried to just run the elixir tests using "mix"; I'll
> remove this and look a little harder at the testing docs for how to run
> tests just against one lib when debugging.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elixir-lang/elixir/pull/3673/files#r38809631>.
>